core/templates/components/question-directives/questions-list/questions-list.component.ts

```diff
@@ -563,27 +564,32 @@ angular.module('oppia').component('questionsList', {
             task: 'add',
             difficulty: DEFAULT_SKILL_DIFFICULTY
           });
+          ctrl.updateSkillLinkage('', false);
         }, function() {
           // Note to developers:
           // This callback is triggered when the Cancel button is
           // clicked. No further action is needed.
         });
       };
 
-      ctrl.updateSkillLinkage = function(commitMsg) {
+      ctrl.updateSkillLinkage = function(commitMsg, updateQuestion) {
```

Comment:
I don't think this should be done this way, it is hiding a lot of context just behind the boolean arg and if is see `updateSkillLinkage`('Update commits', true)` I don't really know what is supposed to do and how will it differ from `updateSkillLinkage`('Update commits', false)` more on this here https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3

I think we can refactor this using the new function `updateSkillLinkageAndQuestions` that will call this method and update the question using then on the Promise returned by this function.

Reply:
Done in #13591